packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -137,8 +577,6 @@ class CupertinoColors {
 ///
 /// * [CupertinoUserInterfaceLevel], an [InheritedWidget] that may affect color
 /// resolution of a `CupertinoDynamicColor`.
-/// * [CupertinoSystemColors], an [InheritedWidget] that exposes system colors
-/// of iOS 13+.
 /// * https://developer.apple.com/documentation/uikit/uicolor/3238042-resolvedcolor.
 @immutable
 class CupertinoDynamicColor extends Color {
```

Comment:
why do we pass 0 up to the constructor instead of just passing _effectiveColor.value and not overriding value?

Reply:
Then the constructor can't be made const (https://github.com/flutter/flutter/pull/39430). IMO the benefit of being able to make a const `CupertinoDynamicColor` and use it in a function as the default parameter outweighs the cost, e.g.
```dart
const CupertinoDynamicColor _kDefaultColor = CupertinoDynamicColor(...);
class ThumbPainter extends CustomPainter {
  const ThumbPainter({
     this.color = _kDefaultColor,
  }) : super(...);
}
```